docfx_yaml/markdown_utils.py

```diff
@@ -301,6 +301,16 @@ def move_markdown_pages(
     # Used to keep track of the index page entry to insert later.
     index_page_entry = None
 
+    list_of_markdown_file_names = [
```

Comment:
The type of this variable is a list. No need to have list in the name. So, you can just call this `markdown_file_names`.

Reply:
Done.